src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.less

```diff
@@ -53,7 +53,7 @@
     .truncate;
   }
   &-owner {
-    font-size: .9em;
+    font-size: 1.0em;
```

Comment:
From what I could see testing this locally, this can be removed without any ill effect.

Reply:
Alright I'll remove that.